optuna/storages/_rdb/alembic/versions/v3.0.0.b.py

```diff
@@ -0,0 +1,48 @@
+"""Change floating point precision and make intermediate_value nullable.
+
+Revision ID: v3.0.0.b
+Revises: v3.0.0.a
+Create Date: 2022-04-27 16:31:42.012666
+
+"""
+from alembic import op
+from sqlalchemy import Float
+
+
+# revision identifiers, used by Alembic.
+revision = "v3.0.0.b"
+down_revision = "v3.0.0.a"
+branch_labels = None
+depends_on = None
+
+FLOAT_PRECISION = 53
+
+
+def upgrade():
+    with op.batch_alter_table("trial_intermediate_values") as batch_op:
+        batch_op.alter_column(
+            "intermediate_value",
+            type_=Float(precision=FLOAT_PRECISION),
+            nullable=True,
+        )
+    with op.batch_alter_table("trial_params") as batch_op:
+        batch_op.alter_column(
+            "param_value",
+            type_=Float(precision=FLOAT_PRECISION),
+            existing_nullable=True,
+        )
```

Comment:
Do you need `param_value` to be nullable?

Reply:
I'm sorry but I couldn't understand your question. I didn't change the `nullable` setting of `param_value`, and I set `existing_nullable=True`.
Do you mean we need to replace `existing_nullable=True` with `nullable=True`?